project/plugins.sbt

```diff
@@ -20,4 +20,11 @@ lazy val build = (project in file(".")).
     }
   )
 
-unmanagedSourceDirectories in Compile += baseDirectory.value.getParentFile / "src" / "main" / "scala"
+// The interplay build uses itself interplay as a plugin. Crazy but true!
+// The following source directories are needed to compile interplay for
+// its own build. Interplay is built with sbt 1 so we include the sbt-1.0
+// source directory too.
+unmanagedSourceDirectories in Compile ++= Seq(
```

Comment:
Missing `baseDirectory.value.getParentFile / "src" / "main" / "scala-sbt-0.13"`?

Reply:
We only need 1.0 here - I've added a new mega-comment!

```scala
// The interplay "meta-build"—the stuff inside the `project`
// directory—uses interplay as a plugin. Since we haven't run the
// interplay "proper build"—the stuff in the root directory—yet we
// can't just say "addSbtPlugin(..."interplay"...) we need to actually
// compile the plugin sources as part of the meta-build.
//
// We do this by adding some of the interplay source directories to
// the meta-build. This means that interplay will be compiled once as
// part of the meta-build (but not published) and then *again* as part
// of the proper build (where it is properly cross-compiled and
// published).
//
// Note that for the meta-build we only need to include the files in
// `scala-sbt-1.0` directory, not the files in the `scala-sbt-0.13`
// directory. This is because the meta-build in the `project`
// directory is only built with sbt 1 (the value set in the
// build.properties file). In a later stage of the sbt build, when we
// do the "proper build" in the root directory, interplay will be
// cross-compiled for both sbt 1 and sbt 0.13. At this stage, for the
// meta-build, we only need sbt 1.
```